### PR TITLE
glossary: remove "data artifact", et al.

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- Remove "(data) artifact" from glossary, as it's a pretty general DS/ML term (not DVC-specific) — per #608
- Other misc. updates around ^
- \+ Small review of glossary term "import stage"

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Sorry, I need to keep these coming because I fix many small things that are out of scope for my main work and they mount up over time.

File: content/docs/api-reference/read.md [hunk 1]
```diff
@@ -79,8 +79,9 @@ These are loaded to memory directly (without using any disc space).
 
 ## Example: Load data from a DVC repository
 
-Any <abbr>data artifact</abbr> hosted online can be loaded directly in your
-Python code with this API. For example, let's say that you want to load and
+Any file tracked tracked in a <abbr>DVC project</abbr> (and
```
Comment (shcheklein) [inline content/docs/api-reference/read.md:82]: tracked tracked

**Review by shcheklein — APPROVED** [review]

File: content/docs/api-reference/read.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/api-reference/read.md]: Yes because you track in Git the data tracked in DVC...

Comment (jorgeorpinel) [reply content/docs/api-reference/read.md]: Jk, fixed in dac21d59.

**Comment by jorgeorpinel** [comment]
🎉

**Pull request merged**